src/model/docker.ts

```diff
@@ -1,8 +1,24 @@
-import { existsSync, mkdirSync } from 'fs';
+import { existsSync, mkdirSync, readFileSync, rmSync } from 'fs';
 import { exec } from '@actions/exec';
 import path from 'path';
 
+const cidfilePath = parameters => {
```

Comment:
would you mind renaming this to `containerIdFilePath` please?

Reply:
Personally, I'm ok with the current name since docker's own flag is `--cidfile`, but I agree it makes it easier to understand what the concept is.

Reply:
I'm happy to write a few comments to document why it is doing what it is doing, but personally I'd rather keep it `cidfile` so it is searchable. 
I've started with `containerIDPath`, but in the end it read a bit wrong to refer to the same concept with multiple names.

Reply:
If you just change this method into `containerIdFilePath`, the rest of the code will also be clear. And you wouldn't need comments, because your code is self descriptive. Note that we don't use ID in both caps but only the first letter of acronyms as caps (camelCase).

Reply:
Addressed.